lib/property_sets/delegator.rb

```diff
@@ -21,10 +21,17 @@ def delegate_to_property_set(setname, mappings)
         raise "Second argument must be a Hash" unless mappings.is_a?(Hash)
 
         mappings.each do |old_attr, new_attr|
+          if ActiveRecord.version >= Gem::Version.new('5.0')
+            attribute old_attr, ActiveModel::Type::Value.new
```

Comment:
What does this do?

Reply:
AR 5 requires that the type be declared for the attribute. This is the generic "any" type because we don't have additional information.

Reply:
We need to tell ActiveRecord to track this attribute with the `attribute` method. Relevant PR and answer here https://github.com/rails/rails/issues/33753#issuecomment-417110000

Reply:
👍